packages/react/src/components/Tabs/next/Tabs.stories.js

```diff
@@ -106,32 +158,52 @@ export const IconOnly = (args) => (
       </IconTab>
     </TabList>
     <TabPanels>
-      <TabPanel>Tab Panel 1</TabPanel>
-      <TabPanel>Tab Panel 2</TabPanel>
-      <TabPanel>Tab Panel 3</TabPanel>
+      <TabPanel>
+        <p>{tabPanelEnums.tab1.panelText}</p>
+      </TabPanel>
+      <TabPanel>
+        <p>
+          {tabPanelEnums.tab2.panelText}{' '}
+          <Link size="lg">{tabPanelEnums.tab2.linkText}</Link>
+        </p>
+      </TabPanel>
+      <TabPanel>
+        <p>{tabPanelEnums.tab3.panelText}</p>
+      </TabPanel>
     </TabPanels>
   </Tabs>
 );
 
 export const Contained = (args) => (
   <Tabs>
     <TabList aria-label="List of tabs" contained {...args}>
-      <Tab>Tab Label 1</Tab>
-      <Tab>Tab Label 2</Tab>
-      <Tab disabled>Tab Label 3</Tab>
-      <Tab title="Tab Label 4 with a very long long title">
-        Tab Label 4 with a very long long title
+      <Tab>{tabPanelEnums.tab1.linkText}</Tab>
```

Comment:
The first tab has no text in the deploy preview

<img width="1356" alt="image" src="https://user-images.githubusercontent.com/3360588/171953618-4f543dc0-f398-4673-83eb-44b6bd1de92e.png">

Maybe it's just this?
```suggestion
      <Tab>{tabPanelEnums.tab1.labelText}</Tab>
```